packages/strapi-helper-plugin/package.json

```diff
@@ -30,7 +30,6 @@
     "add-asset-html-webpack-plugin": "^2.1.2",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
-    "babel-eslint": "^7.2.3",
```

Comment:
I had a problem with `babel-plugin-transform-react-remove-prop-types@0.4.14`, is much work update also this plugins? 😖

Reply:
@maturanomx package updated!